AndyDorokhin/HW10/extras/classes.py

```diff
@@ -0,0 +1,22 @@
+""" 
+    Implement a function class_name_changer(cls, new_name) that changes the name of the class cls to new_name.
+    
+    Check for empty name  - Expected an error
+    Check for name starting with lower case  - Expected an error
+    Check for name with illegal chars - Expected an error
+    Check for name starts with a number - Expected an error
+"""
+
```

Comment:
good job
look at more optimized version of code

def class_name_changer(cls, new_name):
    if not new_name:
        raise ValueError("Name cannot be empty")
    if not new_name[0].isupper() or not new_name.isalnum():
        raise ValueError("Invalid class name")
    cls.__name__ = new_name